templates/firefox-ios.xliff

```diff
@@ -1250,10 +1254,18 @@
         <source>Show Previous Tab</source>
         <note>Label to display in the Discoverability overlay for keyboard shortcuts</note>
       </trans-unit>
+      <trans-unit id="Inactive Tabs" xml:space="preserve">
+        <source>Inactive Tabs</source>
+        <note>Inactive tabs section title</note>
+      </trans-unit>
       <trans-unit id="Increase text size" xml:space="preserve">
         <source>Increase text size</source>
         <note>Accessibility label for button increasing font size in display settings of reader mode</note>
       </trans-unit>
+      <trans-unit id="Jump Back In" xml:space="preserve">
+        <source>Jump Back In</source>
+        <note>Section title for the Jump Back In section</note>
```

Comment:
Could you add a comment explaining what you're jumping back into?
This might affect translations, depending on the remaining context

Reply:
Also, string ID